openassessment/xblock/submission_mixin.py

```diff
@@ -25,6 +25,21 @@ class SubmissionMixin(object):
     work outside the scope of OpenAssessmentBlock.
 
     """
+    
+    UPLOADABLE_IMAGE_MIME_TYPES = set([
+        'image/gif',
+        'image/jpeg',
```

Comment:
yes, as stated in oa_response.js an image.jpg file is still of type image/jpeg...